components/ww3/bld/build-namelist

```diff
@@ -314,17 +315,29 @@ if ($NML_TYPE eq "ww3") {
 }
 if ($NML_TYPE eq "ww3_grid") {
 
-    if (${WAV_GRID} eq "wQU225EC60to30sp32x36") {
+    if (${WAV_SPEC} eq "sp32x36") { #this option only exists for E3SM V1 grid
```

Comment:
This line was a leftover from my hacky attempt to accomplish a similar thing a long time ago. The current v1 grid obstruction files that are on the public html use 50x36, so we'd want to use that spectral resolution here too.